java/code/src/com/redhat/rhn/frontend/dto/SsmUpgradablePackageListItem.java

```diff
@@ -90,6 +91,24 @@ public void setNumSystems(Long numSystemsIn) {
         this.numSystems = numSystemsIn;
     }
 
+    /**
+     * Gets if {@code installhint(reboot-needed)} flag is set for the package.
+     *
+     * @return True if {@code installhint(reboot-needed)} flag is set.
+     */
+    public boolean isPkgReboot() {
```

Comment:
I couldn't find the hibernate mapping definition for these new `pkgReboot` properties. Aren't those required?

Reply:
This Class is a Dto for Mode Queries. It is not used in Hibernate

Reply:
OK